resources/docs/rest-api/index.md

```diff
@@ -233,6 +233,33 @@ query URLs (intended to be sent to a deployed OPeNDAP service). The name
 that name has carried through here.
 
 
+## Tag association for OUS to return the appropriate OPeNDAP URLS
+As a workable, short-term solution, OUS makes use of CMR tagging functionality
+to convert a granule's archive location to an OPeNDAP location.
+
+To achieve this, we fist need to create a tag cmr.earthdata.nasa.ous.datafile.replace:
```

Comment:
Let's format the tag with backticks: `cmr.earthdata.nasa.ous.datafile.replace`

Reply:
done.